test/tests/rackhd11/test_rackhd11_api_templates.py

```diff
@@ -16,6 +16,9 @@
 from nose.plugins.attrib import attr
 @attr(all=True, regression=True, smoke=True)
 class rackhd11_api_templates(fit_common.unittest.TestCase):
+    def setUp(self):
+        fit_common.rackhdapi("/api/1.1/templates/library/testid", action="delete")
```

Comment:
I'm assuming this is to make sure that the id you are going to use in the tests doesn't already exist.  If so, can you add a comment so it is clear to others editing this test later?